pkg/solana/txm/txm_load_test.go

```diff
@@ -1,6 +1,6 @@
 //go:build integration
 
-package txm_test
+package txm
```

Comment:
Why did we change this package?

Reply:
all the other test files I was working on where under `txm` package and not under `txm_test`. That's why I renamed it. Is there any advantage/reason to have it as `txm_test`?

Reply:
Just curious. Usually the best practice is to use a `_test` package for tests. Think in scenarios we have to test/access private methods or fields, we'll keep it under the same package as the implementation. I guess these tests were probably the only ones that only needed the public methods.

Reply:
fixed